### PR TITLE
some minor fixes

### PR DESCRIPTION
**Pull request by cbielow**
PP-HiRes now gives the user some feedback of which MS-levels were actually picked in `auto-mode`.

```
11:10:22 NOTICE: PeakPickerHiRes of node #4 started. Processing ...
Picked spectra by MS-level:
  MS-level 1: 8259 / 8259
  MS-level 2: 0 / 7482
```

### CODE REVIEW

File: src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp [hunk 1]
```diff
@@ -402,7 +402,14 @@ namespace OpenMS
     pickExperiment(input, output, boundaries_spec, boundaries_chrom, check_spectrum_type);
   }
 
-  void PeakPickerHiRes::pickExperiment(const PeakMap& input, PeakMap& output, 
+  struct SpectraPickInfo
+  {
+    int picked{0}; //< number of picked spectra
```
Comment (timosachsenberg) [inline src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp]: why int?

File: src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp [hunk 2]
```diff
@@ -445,18 +457,18 @@ namespace OpenMS
         else
         {
           std::vector<PeakBoundary> boundaries_s; // peak boundaries of a single spectrum
-
-                                                  // determine type of spectral data (profile or centroided)
-          SpectrumSettings::SpectrumType spectrum_type = input[scan_idx].getType();
-
+          SpectrumSettings::SpectrumType spectrum_type = input[scan_idx].getType(true);
```
Comment (timosachsenberg) [inline src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp]: Maybe add in comment what the true means

File: src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp [hunk 3]
```diff
@@ -402,7 +402,14 @@ namespace OpenMS
     pickExperiment(input, output, boundaries_spec, boundaries_chrom, check_spectrum_type);
   }
 
-  void PeakPickerHiRes::pickExperiment(const PeakMap& input, PeakMap& output, 
+  struct SpectraPickInfo
+  {
+    uint32_t picked{0}; //< number of picked spectra
```
Comment (hroest) [inline src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp]:
```suggestion
    uint32_t picked{0}; ///< number of picked spectra
```

File: src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp [hunk 4]
```diff
@@ -402,7 +402,14 @@ namespace OpenMS
...
+    uint32_t total{0}; //< overall number of spectra
```
Comment (hroest) [inline src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp]:
```suggestion
    uint32_t total{0}; ///< overall number of spectra
```

Comment (cbielow) [reply src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakPickerHiRes.cpp]: good catch... I always get this wrong...

**Pull request merged**